### PR TITLE
277035 Business CPH GraphQL Resolver & Mocks

### PR DESCRIPTION
**Pull request by hadidefra**
* Merged two legacy RESTful API Endpoints to an existing single query resolver `business` .
   *  `{{rpp-api}}/SitiAgriApi/cph/organisation/{orgId}/cph-numbers`
   * `{{rpp-api}}/SitiAgriApi/cph/organisation/{orgId}/cph-numbers/{cphNumber}`
 * Created a new GraphQL Types where coordinates are grouped and named under `x` & `y`
 * Create a new GraphQL type for `cph` field to have details of both API endpoints
 * Created two new mock endpoints for Integration tests
 * Updated `business` resolver integration tests to include assertion for new field `cph`
 * Created a new Unit Tests for CPH Details (info) Transformation

### CODE REVIEW

File: app/graphql/resolvers/business/query.js [hunk 1]
```diff
@@ -22,3 +23,13 @@ export const Query = {
     return transformOrganisationCSApplicationToBusinessApplications(response.applications)
   }
 }
+
+export const CPH = async ({ id }, _, { dataSources }) => {
```
Comment (jack-mccabe-defra) [inline app/graphql/resolvers/business/query.js:27]: Could you move these into a separate file cph.js?
That would match the pattern we have elsewhere

**Review by jack-mccabe-defra — CHANGES_REQUESTED** [review]
Could you move cph resolvers to a separate file please

File: app/graphql/resolvers/business/query.js (hunk 1 above)
Comment (hadidefra) [reply app/graphql/resolvers/business/query.js]: Absolutely, please see commit: 3906058

File: test/integration/graphql/business.test.js [hunk 2]
```diff
@@ -177,3 +177,49 @@ describe('Query.business.land', () => {
...
+      `,
+      schema,
+      contextValue: fakeContext
+    })
+
+    expect(result).toEqual({
+      data: {
+        business: {
+          cph: [{
+            number: expect.any(String),
```
Comment (jack-mccabe-defra) [inline test/integration/graphql/business.test.js:210]: Can you match how the other integration tests are done so we are checking against actual values and not just types

File: test/transformers/rural-payments-portal/business-cph.test.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        cphNumber: '43/060/0025',
+        parcelNumbers: [
+          'SP2936 2318'
+        ]
+      }])).toEqual([{
+        number: '43/060/0025',
+        parcelNumbers: [
+          'SP2936 2318'
+        ],
+        coordinate: expect.any(Function),
```
Comment (jack-mccabe-defra) [inline test/transformers/rural-payments-portal/business-cph.test.js:26]: Same here

File: test/integration/graphql/business.test.js (hunk 2 above)
Comment (hadidefra) [reply test/integration/graphql/business.test.js]: Please see latest commit, it's done now

**Review by jack-mccabe-defra — APPROVED** [review]

**Pull request merged**